dallinger/command_line/ec2.py

```diff
@@ -100,18 +100,16 @@ def list__instance_types(ctx, region):
 @click.option("--region", default=None, help="Region name")
 @click.option("--type", default="m5.xlarge", help="Instance type")
 @click.option("--storage", default=32, type=int, help="Storage in GB; default is 32 GB")
-@click.option(
-    "--pem", default="dallinger", help="PEM file name; default is dallinger.pem"
-)
+@click.option("--pem", default=None, help="PEM file name; default is dallinger.pem")
```

Comment:
Is the help message still correct if `default=None` now?

Reply:
I updated it based on Peter's feedback